apps/api/src/domain/models/user-management/group/commands/create-group/__snapshots__/create-group.command.integration.spec.ts.snap

```diff
@@ -5,6 +5,7 @@ CoscradUserGroup {
   "description": "this group is for teachers from community schools",
   "eventHistory": [
     {
+      "id": "41fb2d7f-c483-4e09-a1f0-e9909a6b0002",
```

Comment:
Wonder why this comes through now? It’s a `getter` that is redundant with the meta property. Let's look into this more.